sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -19,15 +19,18 @@
     <SupportedPlatform Include="browser" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
-    <PackageReference Include="System.ClientModel" />
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0' ">
```

Comment:
Is this an interim step before moving the "strip polyfill packages" step in the build targets?

Reply:
I'll take this out - was thinking this might be an easy win but can come back to the build targets